src/validators/block_id/mod.rs

```diff
@@ -103,26 +103,40 @@ pub fn display_current_validators_info(
     );
 
     let mut table = Table::new();
-    table.set_titles(prettytable::row![Fg=>"Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected"]);
+    table.set_titles(prettytable::row![Fg=>"#", "Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected", "Endorsements produced", "Endorsements expected"]);
 
-    for validator in &current_validators {
+    for (index, validator) in current_validators.into_iter().enumerate() {
         let online = if validator.num_expected_blocks + validator.num_expected_chunks == 0 {
+            if validator.num_expected_endorsements == 0 {
+                "NaN".to_string()
```

Comment:
I'd report it as "100%" online or - or "Not yet available"

Reply:
Done